support-frontend/assets/pages/digital-subscriber-checkout/setup/setUpRedux.ts

```diff
@@ -76,7 +76,7 @@ function getInitialContributionType(
 	);
 	return defaultContributionType
 		? defaultContributionType.contributionType
-		: contributionTypes[countryGroupId][0].contributionType;
+		: contributionTypes[countryGroupId][0]?.contributionType ?? 'ONE_OFF';
```

Comment:
Are we sure this isn't changing behaviour?

Reply:
It has some kind of fallback so in practice I think it's always populated, but we could do an `as` instead to definitely not change runtime behaviour

Reply:
I'm sure it's fine, up to you which you go for.